docs/semgrep-cloud-platform/user-management.md

```diff
@@ -157,5 +157,24 @@ To **change a user's role**:
 You cannot change your own role.
 :::
 
+## Appendix: Token scopes
+
+When creating a new token, you are able to set a token's **scope**. Token scopes enable you to limit or grant permissions as necessary. Tokens are also autogenerated with appropriate scopes by Semgrep Cloud Platform based on the user's role or when onboarding a repository.
+
+The following table displays token scopes and their permissions:
+
+| Token scope | Send findings from a remote repository | Send findings from a local repository |  Send PR or MR comments | Connect to Semgrep API |
+| ----------- | ------- | --- | --| -- |
+| Agent (CI)  |   ✔️  Yes    |  ✔️ Yes  | ✔️  Yes [SME to confirm] | ✔️  Yes  |
+| Web API     |  ❌ No    |  ❌ No | ✔️  Yes [SME to confirm] | ✔️ Yes  |
+| Member      |    ❌ No   |  ✔️ Yes  | ❌ No |❌ No |
+
+The following table displays typical uses for token scopes:
+
+| Token scope |Typical uses |
+| ----------- | ----------- |
+| Agent (CI)  | Autogenerated by Semgrep Cloud Platform when onboarding (adding) a repository to Semgrep Cloud Platform. |
```

Comment:
When an admin generates a token from their CLI, what scope is that token?

Reply:
Tokens are generated using the scope of the user so if they are an admin it will be not a member token.

Reply:
Got it. I checked, everything I ever made is Agent, so it's all agent-scoped if no rbac or if admin. Resolving.